src/components/DefaultValidators.tsx

```diff
@@ -0,0 +1,58 @@
+import { defaultMultisigConfigs } from '@hyperlane-xyz/sdk';
+import { chainMetadata } from '@hyperlane-xyz/registry';
+import { useAbacusWorksChainNames } from '../utils/registry';
+import { camelToTitle, Environment } from './AddressTable';
+
+type DefaultValidatorsProps = {
+  environment: Environment;
+};
+
+export default function DefaultValidators({ environment }: DefaultValidatorsProps) {
+  const chainNames = useAbacusWorksChainNames(environment !== "mainnet");
+
+  return (
+    <>
+      {chainNames.map((chain) => {
+        const config = defaultMultisigConfigs[chain];
+        if (!config) return null;
+
+        const { name, displayName, domainId } = chainMetadata[chain];
+
+        return (
+          <div id={chain} key={chain}>
+            <h2 className="mb-4">
+              <a
+                href={`#${chain}`}
+                className="!text-inherit no-underline hover:no-underline"
+                style={{ color: 'inherit' }}
```

Comment:
Why do you need this when you already have text-inherit?

Reply:
`text-inherit` didn't do anything for me, the chain headings were still staying blue. turns out the only one I needed to keep the headings white was `color: inherit`